src/internal/pfsdb/model.go

```diff
@@ -84,11 +85,24 @@ func parseBranches(branchInfos string) ([]*pfs.Branch, error) {
 	return branches, nil
 }
 
-// Commit is a row in the pfs.commits table.
 type Commit struct {
-	ID          CommitID `db:"id"`
-	Repo        Repo     `db:"repo"`
-	CommitSetID string   `db:"commit_set_id"`
+	RowID          CommitID      `db:"int_id"`
```

Comment:
I think it's better to just use `ID` here.

```suggestion
	ID CommitID      `db:"int_id"`
```